pkg/debug/Dockerfile.in

```diff
@@ -37,6 +38,41 @@ RUN for patch in fix-musl-sc_long_bit.patch wrapper-for-basename.patch 155652295
  make -C src RPM_OPT_FLAGS=-DNONLS static &&\
  cp src/lshw-static /lshw && strip /lshw
 
+#FROM intel/oneapi-basekit as vtune-source
```

Comment:
do we need to keep this comment?

Reply:
We might as well skip it - the binary version depends on a full glibc

Reply:
Yeah -- in that case -- let make a separate container that is specifically there for VTune purposes -- lets not pollute debug one @temerkhanov

Reply:
That is the binary agent version which depends on glibc (and pulls a really huge container during build). The one added by this patch is built from source and is OK with musl